platform/email/platform-email-impl/src/main/java/org/codice/ddf/platform/email/impl/SmtpClientImpl.java

```diff
@@ -92,7 +99,7 @@ public void setUserName(@Nullable String userName) {
    * @param password the password for the email server
    */
   public void setPassword(@Nullable String password) {
-    this.password = password;
+    this.password = encryptionService.decryptValue(password);
```

Comment:
❓ Do we want to keep the password encrypted in-memory and only decrypt on use?

Reply:
I assumed that's how this worked already - is it not?

Reply:
why? isn't that actually a safer approach?

Reply:
where does the decryption key live

Reply:
Someone from the @codice/security team could probably help. Is it only accessible here? https://github.com/codice/ddf/blob/2.19.x/platform/email/platform-email-impl/src/main/java/org/codice/ddf/platform/email/impl/SmtpClientImpl.java#L162
If so then possible could decrypt it there so it isn't in memory. Not sure if that is better.

Reply:
The decryption key lives in `etc/ketsets` which is protected by the security policy.

https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/security/default.policy#L234-L235